packages/core/src/scratchorg/pool/PoolFetchImpl.ts

```diff
@@ -131,6 +135,7 @@ export default class PoolFetchImpl extends PoolBaseImpl {
                 let allocateSO = await new ScratchOrgInfoAssigner(this.hubOrg).setScratchOrgInfo({
                     Id: element.Id,
                     Allocation_status__c: 'Allocate',
+                    ValidationId__c: this.validationId
```

Comment:
Looks like this will break in orgs where this field is yet to be deployed, we need to handle this

Reply:
Do we need to upgrade the **sfpower-scratchorg-pool** to a new version 2.1 to reflect this new `ValidationId__c` field or are we adding error handling to skip depending on this field?  We can draft standard manual updates as well to that field in the repo for the **sfpower-scratchorg-pool**

Reply:
Thank you, I will check that